src/bin/wasmer.rs

```diff
@@ -80,12 +81,16 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
         webassembly::InstanceABI::None
     };
 
+    let emscripten_globals = wasmer_emscripten::EmscriptenGlobals::new();
+
     let import_object = if abi == webassembly::InstanceABI::Emscripten {
-        wasmer_emscripten::generate_emscripten_env()
+        wasmer_emscripten::generate_emscripten_env(&emscripten_globals)
     } else {
-        wasmer_runtime::Imports::new()
+        wasmer_runtime::import::Imports::new()
     };
 
+    let import_object = Rc::new(import_object);
```

Comment:
Imports don't use `Rc` anymore, but this is an easy fix once we get the kinks shaken out